test/helpers/fixtures.go

```diff
@@ -0,0 +1,20 @@
+package helpers
+
+import (
+	"path"
+	"runtime"
+)
+
+var fixturesFolder = "" // nolint
```

Comment:
Why this nolint?

Reply:
Because `golangci-lint` doesn't like global variables, but this is a corner case. I could put `// nolint:gochecknoglobals` if that helps.

Reply:
Changed and rebased